app/uk/gov/hmrc/helptosavefrontend/views/register/missing_user_info.scala.html

```diff
@@ -22,20 +22,26 @@
 
 @uk.gov.hmrc.helptosavefrontend.views.html.main_template(title = Messages("hts.register.missinguserinfo.header.page.title"), bodyClasses = None) {
 
-    <div class="grid-layout grid-layout--stacked">
-        <div class="grid-layout__column grid-layout__column--2-3">
-            <header class="page-header text">
-                <h2>@Messages("hts.register.missinguserinfo.header")</h2>
-            </header>
-
-            <ul class="bullets">
-            @for(info <- missingInfos) {
-                <li>@info</li>
-            }
-            </ul>
-
-            <p>@Html(Messages("hts.register.missinguserinfo.redirect.personal.account", personalAccountUrl))</p>
-            <p class="margin-bottom">@Html(Messages("hts.register.missinguserinfo.contact.hmrc", "https://www.gov.uk/contact-hmrc"))</p>
-        </div>
-    </div>
+
+    <header class="page-header text">
+        <h2>@Messages("hts.register.missinguserinfo.header")</h2>
+    </header>
+
+    <p>@Messages("hts.register.missinguserinfo.p1")</p>
+
+    <ul class="bullets">
+    @for(info <- missingInfos) {
+        <li>@info</li>
```

Comment:
will this do a `toString` on the `MissingUserInfo`? Is this what we want or do we want some sort of mapping from each `MissingUserInfo` to a string we want to show? (see https://github.com/hmrc/help-to-save-frontend/blob/master/app/uk/gov/hmrc/helptosavefrontend/models/userinfo/MissingUserInfo.scala#L21)